.gitignore

```diff
@@ -22,25 +22,3 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-
```

Comment:
Nice, this was my rouge changes from yesterday. Great that you removed them!

Reply:
alright, where did they come from? 🤔 didnt understand how they ended up here after my rebase

Reply:
It was when I was trying out amplify deployments using the amplify cli. It added some gitignore rules and they had to be merged to main inorder to deploy